app/models/hackney/repair_request.rb

```diff
@@ -19,9 +24,13 @@ def self.find(repair_request_reference)
     build(response)
   end
 
-  # TODO: improve naming
-  def self.build(attributes)
-    new(attributes_from_api(attributes))
+  # FIXME: bad naming
+  def self.build(api_attributes)
+    new_from_api(api_attributes)
```

Comment:
You may as well get rid of `::new_from_api` and use `new(attributes_from_api(api_attributes))` directly

Reply:
In the end what I want is to get rid of this `self.build()` method, as it seems like it does the same thing as *ActiveRecord*, but doesn't. So what I'm doing is just much aliasing it while I don't refactor the rest of the app.

Does that make sense?